src/js/tests/newsletter-subscription.test.js

```diff
@@ -91,3 +91,60 @@ test('Happy path', async () => {
     '<b>Thank you. Please check your email to confirm your subscription.</b>',
   );
 });
+
+test('IE11 manual fallback', async () => {
+  // Testing that the overall behaviour still works with IE11, which has
```

Comment:
micro-nit: I'd move that comment above the test which makes it look more JSonic than Pythonic 🦔 🐍